projects.rb

```diff
@@ -1,44 +1,88 @@
 class Project
   extend Forwardable
-  def_delegators :@project, :name, :id, :html_url
+
+  PROJECT_BATCHES =
```

Comment:
not sure that works with the logic? I'm basically loading everything out of the cache and then just updating the ones that are in the next batch to be updated